fighthealthinsurance/views.py

```
@@ -2799,7 +2936,446 @@ def get_context_data(self, **kwargs):
+        json_ld = {
+            "@context": "https://schema.org",
+            "@type": "DefinedTermSet",
+            "name": "Health Insurance & Appeals Glossary",
+            "description": (
+                "Plain-language definitions of health insurance and appeals terms "
+                "to help patients understand and fight coverage denials."
+            ),
+            "url": canonical_url,
+            "hasDefinedTerm": defined_terms,
+        }
+        context["json_ld"] = mark_safe(json.dumps(json_ld))
```

Ruff (0.15.20)

[error] Use of `mark_safe` may expose cross-site scripting vulnerabilities

(S308)